src/Migrate/docs/Set-AzMigrateHCIServerReplication.md

```diff
@@ -0,0 +1,275 @@
+---
+external help file:
+Module Name: Az.Migrate
+online version: https://learn.microsoft.com/powershell/module/az.migrate/set-azmigratehciserverreplication
+schema: 2.0.0
+---
+
+# Set-AzMigrateHCIServerReplication
+
+## SYNOPSIS
+Updates the target properties for the replicating server.
+
+## SYNTAX
+
+```
+Set-AzMigrateHCIServerReplication -TargetObjectID <String>
```

Comment:
We call PUT to update servicer replication on the backend, which I believe corresponds to Set instead of Update. I read that Update is for PATCH. Please correct me if wrong.

Reply:
Yes, you are right. In our cmdlet design best design, that's the way to distinguish Update and Set. But for generated cmdlet, it's more common to use update as verb. Anyway, both are accepted for me.